dist/victory-util.js

```diff
@@ -7,7 +7,7 @@
 		exports["VictoryUtil"] = factory(require("react"));
 	else
 		root["VictoryUtil"] = factory(root["React"]);
-})(this, function(__WEBPACK_EXTERNAL_MODULE_145__) {
```

Comment:
Oops, didn't mean to check this in :flushed:. As discussed in chat it shouldn't matter in this case since we will do a new release build for these changes(?) but I will make sure not to in the future.